modules/introduction/partials/new-features-76.adoc

```diff
@@ -35,6 +41,11 @@ This feature gives you greater flexibility when authenticating users.
 For example, you can use a regular expression to map the domain name in an email address to an LDAP organization. 
 See xref:manage:manage-security/configure-ldap.adoc#ldap-advanced-mapping[Advanced Query] under xref:manage:manage-security/configure-ldap.adoc#enable-ldap-user-authentication[User Authentication Enablement].
 
+* The Couchbase Server Web Console now supports using _Structured Authentication Markup Language_ (SAML) for authentication. 
```

Comment:
I know italics are common in the Learn and Manage sections; just to note that italics are out of favor in the current style guide. Not a blocker for this PR though.